core/src/main/scala/better/files/Cmds.scala

```diff
@@ -70,9 +70,19 @@ object Cmds {
 
   def unzip(zipFile: File)(destination: File): File = zipFile unzipTo destination
 
-  def zip(files: File*)(destination: File): File = returning(destination) {
+  def zip(files: File*)(destination: File, compression: Boolean = true, compressionLevel: Int = 9): File = returning(destination) {
     for {
-      output <- new ZipOutputStream(destination.newOutputStream).autoClosed
+      output <- {
+        val out = new ZipOutputStream(destination.newOutputStream)
+
```

Comment:
It would be nice to put the link to the Stackoverflow answer as a code comment here..